src/nd.h

```diff
@@ -18,142 +18,149 @@
 using namespace godot;
 
 class nd : public Object {
-	GDCLASS(nd, Object)
+GDCLASS(nd, Object)
 
 protected:
-	static void _bind_methods();
+    static void _bind_methods();
```

Comment:
You are adding a lot of changes that are exchanging tab indentation with space indentation.
It's a bit odd that your code editor would not auto-detect the indentation, but in any case, you watch out to avoid committing changes like this in the future.

Reply:
Actually, this one seems to be more my fault. A lot of the files are currently mixing tabs and spaces. I'll fix that immediately!